hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -75,21 +77,23 @@ public TableMetadata refresh() {
       String tableType = table.getParameters().get(TABLE_TYPE_PROP);
 
       if (tableType == null || !tableType.equalsIgnoreCase(ICEBERG_TABLE_TYPE_VALUE)) {
-        throw new IllegalArgumentException(format("Invalid tableName, not Iceberg: %s.%s", database, table));
+        throw new IllegalArgumentException(String.format("Invalid tableName, not Iceberg: %s.%s", database, table));
       }
 
       metadataLocation = table.getParameters().get(METADATA_LOCATION_PROP);
       if (metadataLocation == null) {
-        throw new IllegalArgumentException(format("%s.%s is missing %s property", database, tableName, METADATA_LOCATION_PROP));
+        String errMsg = String.format("%s.%s is missing %s property", database, tableName, METADATA_LOCATION_PROP);
+        throw new IllegalArgumentException(errMsg);
       }
 
     } catch (NoSuchObjectException e) {
       if (currentMetadataLocation() != null) {
-        throw new NoSuchTableException(format("No such table: %s.%s", database, tableName));
+        throw new NoSuchTableException(String.format("No such table: %s.%s", database, tableName));
       }
 
     } catch (TException e) {
-      throw new RuntimeException(format("Failed to get table info from metastore %s.%s", database, tableName), e);
+      String errMsg = String.format("Failed to get table info from metastore %s.%s", database, tableName);
+      throw new RuntimeException(errMsg, e);
```

Comment:
We have a RuntimeMetaException which supports formatting, should we create a RuntimeTException to supporting formatting ?

Reply:
Good ideas. I'm fine either way with this. It would be nice to clean this up now, but I am fine with doing it in a separate commit since this is supposed to clean up formatting, not make substantive changes.